module-4/backend/week_three.md

```diff
@@ -5,15 +5,46 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+  Node by definition is a Server Side Enviroment for JavaScript, written in vanilla JS.  I commonly mistake it for a server side language similar to Ruby but that would be incorrect as it just transforms a client-side based language into a server-side capable language?
```

Comment:
yup, it's not a server side language because node isn't a language itself. node is written in javascript, but node is a server side environment